docs/ides/metals.md

```diff
@@ -4,28 +4,33 @@ title: Metals
 sidebar_label: Metals
 ---
 
-Metals is a work-in-progress language server for Scala that supports a diverse number of text
-editors such as Visual Studio Code, `vim`, Sublime Text and Atom.
+Metals is a language server for Scala that supports a diverse number of text
+editors such as Visual Studio Code, Vim, Emacs, Eclipse, Sublime Text and Atom.
 
 |                              | Build import | Compile        | Test                  | Run                  | Debug                 |
 | ---------------------------- | ------------ | -------------- | --------------------- | -------------------- | --------------------- |
-| **Metals**                   | ✅           | ✅             | ✅                   | ✅                   | ✅                   |
+| **Metals**                   | ✅           | ✅             | ✅                    | ✅                   | ✅                    |
 
-At the moment, the BSP integration only supports build import and compilation (with compiler
-diagnostics integrated in the editor). This functionality is enough to provide a fluent developer
-experience. Support for test is planned.
+
+The BSP integration in metals supports build import, compilation, Test, Run, and
+Debug functionality.
 
 ![Gif of compiler diagnostics in metals](assets/metals-compiler-diagnostics.gif)
 
 ## Installation requirements
 
-1. At least Bloop v1.1.0
-1. A build with at least one project in it
-1. You use a [build tool supported by Bloop](build-tools/overview.md)
+If you don't have Bloop installed locally, Metals will automatically start a
```

Comment:
Everything from here to the rest is just updated information about how Metals is using bloop. For example, it's not necessary for the user to manually install Bloop to use it with metals or to manually add the sbt plugin. I wanted to clarify that a bit in the docs.